src/ketrew_backend/plugin.ml

```diff
@@ -1,15 +1,17 @@
 
 open Coclobas
 open Internal_pervasives
+let (//) = Filename.concat
 
-let name = "coclobas-kube"
+let name = "coclobas"
 
 module Run_parameters = struct
 
   type created = {
     client: Client.t;
-    specification: Kube_job.Specification.t [@main];
-    program: Ketrew_pure.Program.t option; (* Kept for display purposes *)
+    specification: Job.Specification.t [@main];
+    program: Ketrew_pure.Program.t option;
+    playground_path: string option;
```

Comment:
Playground goes by "ketrew"'s definition: a directory to put scripts and tmp-files.

There is an exception only in the case where you ask of a `` `Local_docker``
job, **with** a `Program.t`, and no playground.
All other cases are fine without a playground
(`` `Local_docker`` + one command; Kubernetes + `Program.t`; … etc.)